database/factories/ModelFactory.php

```diff
@@ -9,34 +9,34 @@
 | you a convenient way to create models for testing and seeding your
 | database. Just tell the factory how a default model should look.
 |
-*/
+ */
 
 /** @var \Illuminate\Database\Eloquent\Factory $factory */
 $factory->define(App\Models\SUser::class, function (Faker\Generator $faker) {
     static $password;
 
     return [
-        'name' => $faker->name,
-        'email' => $faker->unique()->safeEmail,
-        'password' => $password ?: $password = bcrypt('secret'),
+        'name'           => $faker->name,
```

Comment:
Bad practice to allign like this; if a new item is added and it has a longer key, all other lines need an update (= diff state in PR) to.